app/routers/event.py

```diff
@@ -84,6 +90,60 @@ async def eventedit(request: Request) -> Response:
     return templates.TemplateResponse("eventedit.html", {"request": request})
 
 
+@router.get("/edit/view_countries", include_in_schema=False)
+async def choose_country(request: Request,
+                         session=Depends(get_db)) -> Response:
+    """
+    Displays the list of all countries name from "Country" table.
+    """
+    countries_names = get_all_countries_names(session=session)
+    return templates.TemplateResponse("eventedit.html",
+                                      {"request": request,
+                                       "countries_names": countries_names})
+
+
+@router.post("/edit/view_countries", include_in_schema=False)
+async def check_country_time(request: Request,
+                             session=Depends(get_db)) -> Response:
+    """
+    Displays datalist of all countries name from "Country" table.
+    By clicking "Check time" the converted meeting time is displayed.
+    If the user's ip is not recognized, an error message will appear.
+    ** temporarily using a random israeli ip address instead **
+    """
+    # TODO: define ip as request.client.host
+    # ip = request.client.host
+    random_israeli_ip_for_now = '82.166.236.10'
+    ip = random_israeli_ip_for_now
+    match = geolite2.lookup(ip)
```

Comment:
This module sends synchronic request to geolite.
That's a problem because it can lead to huge slowdowns, as it blocks any other task from happening.
You can use [the async version](https://github.com/maxmind/GeoIP2-python#async-web-service-example) or transform it to the JS and use location API

Reply:
I transformed things to JS